### DISCUSSION TITLE
isCustomElement to use Shoelace components

### DISCUSSION THREAD

**Discussion by graceyoung-ww** [FranckFreiburger/vue3-sfc-loader#153] [Q&A]
I'm doing some tests with Shoelace 2.0, and I'm able to get it working just fine when the `<sl-` component is in the root app (main.html), but the `app.config.compilerOptions.isCustomElement = (tag) => tag.startsWith('sl-')` is being ignored for any components loaded using vue3-sfc-loader.

This results in the components working just fine, but a bunch of runtime compiler warnings appearing in console.

Is there a way to get the `isCustomElement` method to be honoured by vue3-sfc-loader?

**Comment by FranckFreiburger** [comment]
https://github.com/FranckFreiburger/vue3-sfc-loader/commit/7a88a1c484b7ee9b16af7ae0531631fc0876ff95
